audiocall/ConnectionService.MD

```diff
@@ -100,24 +98,24 @@ call?.hold(hold, object : ICallCompletionHandler {
 ```
 
 Please note:
-- On devices running Android API level 28 or higher, when an incoming PSTN call comes and is answered by the user, the VoIP ongoing call will be put on hold by `TelecomManager` and could be resumed after the PSTN call ends.
+- On devices running Android API level 28 or higher, when an incoming PSTN call is answered by the user, the VoIP ongoing call is put on hold by `TelecomManager` and can be resumed after the PSTN call ends.
 - On devices running Android API level 27 or lower, only one app can maintain an ongoing call at any given time. This constraint means that the ongoing call will be terminated.
 
 <img src="../screenshots/schemes/connection_scheme_ongoing_call.png">
 
 ### End ongoing call
-The ongoing call may be ended by the user or the remote party. 
+Either a user or a remote party can end an ongoing call. 
 
-If the user ends the call, the application should call `ICall.hangup()` API and Voximplant Android SDK will invoke `ICallListener.onCallDisconnected()` event. 
+If the user ends the call, the application should call `ICall.hangup()` API, so the Voximplant Android SDK triggers the `ICallListener.onCallDisconnected()` event. 
 
-If the remote party has ended the call, Voximplant Android SDK will inform the application by invoking `ICallListener.onCallDisconnected()` event.
+If the remote party ends the call, the Voximplant Android SDK informs the application about it by triggering the `ICallListener.onCallDisconnected()` event.
 
-Upon receiving `ICallListener.onCallDisconnected()` event it is reqired:
+After receiving the `ICallListener.onCallDisconnected()` event you need to:
 1. Change the `Connection` status to disconnected via `Connection.setDisconnected(DisconnectCause)`
-2. Destroy the `Connection` via `Connection.destroy()` API to tears down the Connection object.
+2. Destroy the `Connection` via `Connection.destroy()` API to destroy the Connection object.
 
-`DisconnectCause` describes the cause of a disconnected call. This always includes a code describing the generic cause of the disconnect. Optionally, it may include a label and/or description to display to the user. It is the responsibility of the `ConnectionService` to provide localized versions of the label and description. It also may contain a reason for the disconnect, which is intended for logging and not for display to the user.
+`DisconnectCause` describes the cause of a disconnected call. It always includes a code describing the generic cause of the disconnect. Optionally, it may include a label and/or description to be displayed to the user. The `ConnectionService` provides localized versions of the label and description. It also may contain a disconnect reason to be put to logs, not to display to the user.
```

Comment:
this part is fully taken from android API references: https://developer.android.com/reference/android/telecom/DisconnectCause

Reply:
I would change it anyway, some sentences look weird. 
But this is not my final version; now I see a couple of things to correct and I'll let you know when it's done.